corehq/apps/export/static/export/js/export_list.js

```diff
@@ -38,37 +38,55 @@ hqDefine("export/js/export_list", [
     var exportModel = function (options, pageOptions) {
         assertProperties.assert(pageOptions, ['is_deid', 'model_type', 'urls']);
 
-        _.each(['isAutoRebuildEnabled', 'isDailySaved', 'isFeed', 'showLink'], function (key) {
+        _.each(['isAutoRebuildEnabled', 'isDailySaved', 'isFeed', 'isOData', 'showLink'], function (key) {
             options[key] = options[key] || false;
         });
         options.formname = options.formname || '';
 
+        assertProperties.assert(options, [
+            'addedToBulk',
+            'can_edit',
+            'deleteUrl',
+            'description',
+            'downloadUrl',
+            'editUrl',
+            'emailedExport',
+            'exportType',
+            'filters',
+            'formname',
+            'id',
+            'isDeid',
+            'lastBuildDuration',
+            'name',
+            'owner_username',
+            'sharing',
+        ], [
+            'case_type',
+            'isAutoRebuildEnabled',
+            'isDailySaved',
+            'isFeed',
+            'isOData',
+            'showLink',
+        ]);
         assertProperties.assert(pageOptions.urls, ['poll', 'toggleEnabled', 'update']);
 
         var self = ko.mapping.fromJS(options);
-        self.prepareExportError = ko.observable('');
-        self.hasEmailedExport = !!options.emailedExport;
 
-        // Unwrap the values in the EMWF filters, turning them into plain {id: ..., text: ...} objects for use with select2
+        self.showSavedFilters = !!options.filters;
+        if (self.showSavedFilters) {
+            // un-knockoutify case and form filter objects
+            self.filters.emwf_case_filter(options.filters.emwf_case_filter);
+            self.filters.emwf_form_filter(options.filters.emwf_form_filter);
+        }
+
+        self.hasEmailedExport = !!options.emailedExport;
         if (self.hasEmailedExport) {
-            self.emailedExport.filters.emwf_case_filter(_.map(self.emailedExport.filters.emwf_case_filter(), function (mw) {
-                return _.mapObject(mw, function (observable) {
-                    return observable();
-                });
-            }));
-            self.emailedExport.filters.emwf_form_filter(_.map(self.emailedExport.filters.emwf_form_filter(), function (mw) {
-                return _.mapObject(mw, function (observable) {
-                    return observable();
-                });
-            }));
+            self.emailedExport = emailedExportModel(options.emailedExport, pageOptions, self.id(), self.exportType());
         }
 
-        self.justUpdated = ko.computed(function () {
-            if (self.emailedExport.taskStatus === undefined) {
-                return false;
-            }
-            return self.emailedExport.taskStatus.justFinished() && self.emailedExport.taskStatus.success();
-        });
+        if (options.isOData) {
+            self.odataFeedUrl = 'https://placekitten.com';
+        }
```

Comment:
@nickpell this is where the URL is generated

Reply:
Nice placeholder :)